tests/test_bridge.py

```diff
@@ -42,21 +42,21 @@ def udp_broadcast_server():
 
 @patch("logging.Logger.info")
 async def test_stopping_before_started_and_establishing_a_connection_should_write_to_the_info_output(mock_info, mock_callback):
-    bridge = SwitcherBridge(mock_callback)
+    bridge = SwitcherBridge(mock_callback, [1234])
     assert_that(bridge.is_running).is_false()
     await bridge.stop()
-    mock_info.assert_called_with("udp bridge not started")
+    mock_info.assert_called_with("udp bridge on port %s not started", 1234)
 
 
 async def test_bridge_operation_as_a_context_manager(unused_udp_port_factory, mock_callback):
     port = unused_udp_port_factory()
-    async with SwitcherBridge(mock_callback, port) as bridge:
+    async with SwitcherBridge(mock_callback, [port]) as bridge:
```

Comment:
Shouldn't we **also** use multiple ports to make this and other tests in this file better reflect the modification in this PR?
I think we should add tests that sort, or perhaps a better idea will be to parameterize the current test cases...

Reply:
Working on a WIP branch and "adding tests later" is ok, because it's relatively easy to locate missing tests.
However, realizing you need to modify an existing working test by just adding another test case to make it cover another end case,
I believe this is more easily noted before merging to the WIP branch unless, of course, you already took note of this and plan to include it when adding the new tests.
:-)

Reply:
Note my initial https://github.com/TomerFi/aioswitcher/pull/484#discussion_r897784440 :
> Shouldn't we **also** use multiple ports

meaning, adding another test case regardless of failing/passing tests.